src/test/api/entities.test.js

```diff
@@ -249,6 +249,24 @@ describe('entities spec', () => {
 
         expect(result).toEqual(OK_RESPONSE);
       });
+
+      it('pauseSource', async () => {
+        const method = 'Post';
+        mock[`on${method}`](`/api/sources/v3.1/sources/${SOURCE_ID}/pause`).reply(200, OK_RESPONSE);
```

Comment:
Are you sure you want to use mock for async calls?

Reply:
This is a test of the url endpoint.